app/src/main/java/com/pileproject/drive/execution/ExecutionActivity.java

```diff
@@ -135,14 +152,25 @@ public void onClick(View v) {
     public void onStart() {
         super.onStart();
 
-        if (hasBluetoothFunction()) {
-            if (!mBtAdapter.isEnabled()) {
-                startActivityForResult(new Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE),
-                                       REQUEST_ENABLE_BT);
-            } else {
-                connectToDevice();
-            }
+        // TODO: remove bluetooth-dependent code for WifiCommunicator
+        if (!DeployUtils.isOnEmulator() && !BluetoothUtil.hasBluetoothFunction()) {
```

Comment:
How about making an issue for this as a note so that we can remember we have a plan to prepare `WifiCommunicator`?